tests/boundingboxcomputation_tests.ts

```diff
@@ -235,4 +211,35 @@ const BoundingBoxComputationTests = {
   },
 };
 
+//#region  Helper Functions
+function createCanvas(options: TestOptions): RenderContext {
```

Comment:
Same comments about moving to top.

Reply:
> Also, can we move these helpers to the top? It's more readable if dependencies are declared before the they are used.

Yeah, I will move the helper functions up.

I was struggling with making the test files look consistent. For example, I want the `Start()` function to be as close to the top as possible, because I believe it gives a great overview on how many test cases will be run, and what each test case is called.

Some of the test files previously had nested inline callback functions that impaired readability, so I tried to extract those out into helper methods. Since editors and IDEs (e.g., vs code) allow you to quickly follow method invocations (via keyboard shortcut or ctrl+click / cmd+click), I figured it was okay to push the helper functions down.

But yes, I generally agree that hiding helper functions at the bottom is not the best practice. I'll find a way to move them up while keeping the Start() method easy to look for.

Reply:
I see what you're saying about consistency, and I really like the methods you extracted. I think of consistency as a tool for making code more readable. So in this case, the consistency takes away the readability :-)

Reply:
Got it. I'll move the helpers up. Will ping you when it's ready for a second review.